controllers/actions.github.com/resourcebuilder.go

```diff
@@ -200,6 +200,12 @@ func (b *resourceBuilder) newScaleSetListenerPod(autoscalingListener *v1alpha1.A
 				Command: []string{
 					"/github-runnerscaleset-listener",
 				},
+				Ports: []corev1.ContainerPort{
```

Comment:
How should we enable settings?
There is a `metrics-addr` flag there. What I would do is, if `metrics-addr` is set, enable metrics on both the controller and the listener. Otherwise, don't expose metrics.
The only problem is that it takes the default value of `:8080`, so basically, to turn it off, you would have to say: `--metrics-addr=""` explicitly. 
Follow-up question: can we use a constant port for the listener, or should we set it based on the `metrics-addr`?